frontend/src/components/Locations/LocationForm.tsx

```diff
@@ -47,19 +49,25 @@ const LocationForm = ({current, createNew, setLocation, PickupDeliveryObject, se
   const [food, setFoodType] = useState("");
   const [demographic, setDemographic] = useState("");
   const [area, setArea] = useState("");
+  const[input, setInput] = useState("");
 
   const { name } = current;
 
-  const submitPressed = () => {
-    if (createNew === false && active !== ""){
+  const submitPressed = () => 
+  {
+    if(isOtherClicked)
+    {
+      active.push(input);
+    }
```

Comment:
`console.log(`User selections: ${active}`)
```suggestion
    }
```

Reply:
Can you add something like this to make it clear for reviewers what is being submitted? This is also just helpful for future debugging